UI/SidePanel.cpp

```diff
@@ -2535,6 +2542,15 @@ void SidePanel::PlanetPanelContainer::SetPlanets(const std::vector<int>& planet_
         GG::Connect(m_planet_panels.back()->RightClickedSignal,         PlanetRightClickedSignal);
         GG::Connect(m_planet_panels.back()->BuildingRightClickedSignal, BuildingRightClickedSignal);
         GG::Connect(m_planet_panels.back()->ResizedSignal,              &SidePanel::PlanetPanelContainer::RequirePreRender,       this);
+        m_planet_panels.back()->OrderButtonChangedSignal.connect(
+            [this](int planet_id) {
```

Comment:
instead of a new function here, could perhaps use, and slightly extend PlanetPanelContainer::RefreshAllPlanetPanels

https://github.com/freeorion/freeorion/blob/master/UI/SidePanel.cpp#L2715